roles/gpu_operator_wait_deployment/tasks/main.yml

```diff
@@ -65,7 +65,7 @@
     register: has_gpu_feature_discovery_labels
     until:
     - has_gpu_feature_discovery_labels.stdout != ""
-    retries: 10
+    retries: 30
```

Comment:
not directly related, did you have issue with this test?

Reply:
Yes. This failed for me. It looked like the driver build took a bit longer due to the mirror, not sure.